go.mod

```diff
@@ -1,71 +1,68 @@
 module github.com/rancher/remotedialer
 
-go 1.23
+go 1.24.0
 
 require (
 	github.com/gorilla/mux v1.8.1
-	github.com/gorilla/websocket v1.5.3
+	github.com/gorilla/websocket v1.5.4-0.20250319132907-e064f32e3674
```

Comment:
Is this intentional?  It looks like v1.5.3 is still the latest released version for websocket.